packages/css-processor/src/CSSNativeParseRun.ts

```diff
@@ -40,11 +40,20 @@ const extraLongViewStyles: Record<ExtraNativeLongViewStyleKeys, 'block'> = {
   shadowOffset: 'block',
   shadowOpacity: 'block',
   shadowRadius: 'block',
-  testID: 'block',
   tintColor: 'block',
   transformMatrix: 'block',
   translateX: 'block',
-  translateY: 'block'
+  translateY: 'block',
+  borderCurve: 'block',
+  gap: 'block',
+  inset: 'block',
+  insetBlock: 'block',
+  insetInline: 'block',
+  marginBlock: 'block',
+  marginInline: 'block',
+  paddingBlock: 'block',
+  paddingInline: 'block',
+  objectFit: 'block'
```

Comment:
That won't work; this is due to the new react-native version supporting more of these props. But we need to implement the rules for each of these. 

I suggest you instead:

- suppress the error by adding `as never` after the object literal;
- open a ticket to report the issue, I'll work on it;
- add a comment referring to the ticket number, showing this is a momentarily hack.